docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md

```diff
@@ -40,9 +40,21 @@ How to Run Model Optimizer
 
 To convert a model to IR, you can run Model Optimizer by using the following command:
 
-.. code-block:: sh
+.. tab-set::
 
-   mo --input_model INPUT_MODEL
+    .. tab-item:: CLI tool
+       :sync: cli-tool
+
+       .. code-block:: sh
+
+          mo --input_model INPUT_MODEL
+
+    .. tab-item:: MO Python API
+       :sync: mo-python-api
+
+       .. code-block:: sh
+
```

Comment:
Please add following import to all MO Python API examples:

`from openvino.tools.mo import convert_model`